Compilation fails on fedora-release-23 PPC64LE

[10.7.65.195]
```
  CC       ib/dc/base/libuct_la-dc_iface.lo
  CC       ib/dc/base/libuct_la-dc_ep.lo
  CC       ib/dc/verbs/libuct_la-dc_verbs.lo
In file included from ib/dc/verbs/dc_verbs.h:10:0,
                 from ib/dc/verbs/dc_verbs.c:7:
ib/dc/verbs/dc_verbs.c: In function 'uct_dc_verbs_iface_progress':
/var/tmp/OFED_topdir/BUILD/ucx-1.0.2385/src/uct/ib/dc/base/dc_iface.h:120:27: error: array subscript is above array bounds [-Werror=array-bounds]
         if (iface->tx.dcis[i].txqp.qp->qp_num == qp_num) {
                           ^
cc1: all warnings being treated as errors
Makefile:1363: recipe for target 'ib/dc/verbs/libuct_la-dc_verbs.lo' failed
make[2]: *** [ib/dc/verbs/libuct_la-dc_verbs.lo] Error 1
make[2]: Leaving directory '/var/tmp/OFED_topdir/BUILD/ucx-1.0.2385/src/uct'
```

--- a/src/uct/ib/dc/base/dc_iface.h
+++ b/src/uct/ib/dc/base/dc_iface.h
@@ -117,10 +117,11 @@ ucs_status_t uct_dc_iface_fc_handler(uct_rc_iface_t *rc_iface, unsigned qp_num,
  */
 static inline uint8_t uct_dc_iface_dci_find(uct_dc_iface_t *iface, uint32_t qp_num)
 {
-    int i;
+    uct_dc_dci_t *dcis = iface->tx.dcis;
+    int i, ndci = iface->tx.ndci;
 
-    for (i = 0; i < iface->tx.ndci; i++) {
-        if (iface->tx.dcis[i].txqp.qp->qp_num == qp_num) {
+    for (i = 0; i < ndci; i++) {
+        if (dcis[i].txqp.qp->qp_num == qp_num) {
             return i;
         }
     }

